app/src/main/java/app/wefridge/wefridge/datamodel/Item.kt

```diff
@@ -0,0 +1,58 @@
+package app.wefridge.wefridge.datamodel
+
```

Comment:
Just a small thing: Why is the directory called "datamodel" and not "model"?

Reply:
As far as I know there's no specific reason. Do you think "model" would be a better name?

Reply:
I would prefer it